### PR TITLE
V8 compile answer

### PR DESCRIPTION
**Pull request by lucasgouvea**
Adding the V8 compile steps descriptions with a flow diagram.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1555,7 +1555,23 @@ V8 is Google’s open source high-performance JavaScript and WebAssembly engine,
 
 ## Q. ***How V8 compiles JavaScript code?***
 
-*ToDo*
```
Comment (learning-zone) [inline README.md:1558]: This answer can be more optimized. Please refere this blog for more information

https://cabulous.medium.com/how-v8-javascript-engine-works-5393832d80a7

**Review by learning-zone — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (lucasgouvea) [reply README.md:1558]: Sure, I'll describe it in more detail.

Comment (lucasgouvea) [reply README.md:1558]: @learning-zone, 
I've been studying/writing it for a while, and I've come to this. Do you think I can detail it better in any part?

**Comment by learning-zone** [comment]
This answer can be more optimized. Hence closing this PR. 
Thanks.

**Pull request closed without merging**